src/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "icons": [
+    {
+      "src": "./assets/images/logo.png",
+      "type": "image/png",
+      "sizes": "48x48"
+    },
+    {
+      "src": "./assets/images/logo.png",
+      "type": "image/png",
+      "sizes": "96x96"
+    },
+    {
+      "src": "./assets/images/logo.png",
+      "type": "image/png",
+      "sizes": "144x144"
+    },
+    {
+      "src": "./assets/images/logo.png",
+      "type": "image/png",
+      "sizes": "256x256"
+    },
+    {
+      "src": "./assets/images/logo.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": "index.html",
```

Comment:
I think this should be `"."` instead of `"index.html"`. What do you think?

Reply:
@singhpratyush I want to point to the first file, that is `index.html`, and it's the root of the site.

Reply:
But there is no such path as `index.html`. Check [this](https://developer.mozilla.org/en-US/docs/Web/Manifest#start_url.